home/default.nix

```diff
@@ -3,33 +3,42 @@ with lib;
 
 let
 
-  cfg = config.colorschemes.kauz;
+  cfg = config.kauz;
+
+  isFish = cfg.fish.enable;
+  isKitty = cfg.kitty.enable;
+  isTmux = cfg.tmux.enable;
+  isNeovim = cfg.neovim.enable;
 
 in {
 
-  options.colorschemes.kauz = {
-    enable = mkEnableOption "a colorscheme for neovim, kitty, fish and tmux";
+  options.kauz = {
+    fish.enable = mkEnableOption "Whether to enable the colorscheme for Fish";
```

Comment:
The description doesn't mention Kauz. Perhaps something like "Whether to enable the Kauz colorscheme for Kitty", etc?